### PR TITLE
Add some basic support for JFR recording into run target.

### PR DESCRIPTION
**Pull request by MikeNeilson**
## Problem Description

Makes it easier to perform some basic profiling to see where hot spots may be.

## Solution

Added logic to add appropriate JFR activation settings to java options passed into the JDK.

## how you tested the change

Manually, ran with JFR on, performed operations, closed app, opened recording in VisualVM (2.1.8) to see results.

## Where the following done:

- [ ] Tests. Check all that apply:
   - [ ] Unit tests created or modified that run during ant test.
   - [ ] Integration tests created or modified that run during integration testing
         (Formerly called regression tests.)
   - [ ] Test procedure descriptions for manual testing
- [x] Was relevant documentation updated?
- [ ] Were relevant config element (e.g. XML data) updated as appropriate

If you aren't sure leave unchecked and we will help guide you to want needs changing where.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -773,16 +773,45 @@ more information about each.
...
         </chmod>
         <mkdir dir="${build.dir}/run"/>
+        <if>
+            <and>
+                <islessthan arg1="${java.specification.version}" arg2="11"/>
+                <isset property="opendcs.jfr"/>
+            </and>
+            <then>
+                <property name="jfrOpts" value=""/>
+                <echo leve="warning">Can only run Java Flight recorder on JDK 11 or above</echo>
```
Comment (ktarbet) [inline build.xml:783]:
```suggestion
                <echo level="warning">Can only run Java Flight recorder on JDK 11 or above</echo>
```

File: docs/source/dev-docs.rst [hunk 2]
```diff
@@ -154,6 +154,12 @@ the "stage" directory is used as DCSTOOL_HOME and DCSTOOL_USERDIR is the same de
     # to run with the java remote debugger enabled
     ant run -DdebugPort=5006
 
+    # to run with Java Flight Recorder
+    ant run -Dopendcs.jfr
```
Comment (ktarbet) [inline docs/source/dev-docs.rst:158]:
```suggestion
    ant run -Dopendcs.jfr=true
```

File: build.xml [hunk 3]
```diff
@@ -773,16 +773,45 @@ more information about each.
...
+            <then>
+                <property name="jfrOpts" value=""/>
+                <echo level="warning">Can only run Java Flight recorder on JDK 11 or above</echo>
+            </then>
+            <elseif>
+                <isset property="opendcs.jfr"/>
+                <then>
+                    <property name="can_jfr" value="true"/>
+                    <property name="jfrOpts"
+                                value="-XX:StartFlightRecording=disk=true,dumponexit=true,filename=${build.dir}/run/${opendcs.app}.recording.jfr,settings=profile"/>
```
Comment (MikeNeilson) [inline build.xml:790]: I'm also going to change the file name here to include the profile. Sometimes I'm testing several at once.

**Review by ktarbet — APPROVED** [review]

File: docs/source/dev-docs.rst [hunk 4]
```diff
@@ -154,6 +154,12 @@ the "stage" directory is used as DCSTOOL_HOME and DCSTOOL_USERDIR is the same de
     # to run with the java remote debugger enabled
     ant run -DdebugPort=5006
 
+    # to run with Java Flight Recorder
+    ant run -Dopendcs.jfr=true
```
Comment (ktarbet) [inline docs/source/dev-docs.rst:158]:
```suggestion
    ant run -Dopendcs.jfr=true   -Dopendcs.profile=/project1/my-nice.profile
```

Is this how the profile feature would be used?

Comment (MikeNeilson) [reply docs/source/dev-docs.rst:158]: Yep.... apparently I not previously document that with the run target documentation.

Comment (MikeNeilson) [reply docs/source/dev-docs.rst:158]: Made a minor update and typo fix.

**Review by ktarbet — APPROVED** [review]

**Pull request merged**